data/human/human missions.txt

```diff
@@ -6891,3 +6891,335 @@ mission "Returning to Paradise"
 			action
 				payment 120000
 			`You're not sure if you've ever met a more pretentious and entitled family. The girls fought non-stop, but fortunately, the family was satisfied with the luxury accommodations on your ship. Once you land on <planet>, the lady gives you <payment> and the family leave without a word of thanks; they never even tell you their names. Nonetheless, you're happy to be rid of them.`
+
+mission "Adelita 1"
+	name "Exhibition installations to <planet>"
+	description `Transport <tons> of exhibition installations to <destination> for the Adelita collective. Payment is <payment>.`
+	minor
+	cargo "exhibition installations" 5
+	source
+		attributes "paradise"
+		attributes "rich"
+	destination "Moonshake"
+	to offer
+		random < 80
+		has "Expedition to Hope 2: done"
+		"combat rating" > 40
+
+	on offer
+		conversation
+			`Walking on the tarmac of <origin>'s luxurious spaceport, you notice a significant group of people gathered around one of the larger hangars. From the signs nearby that display "The Evacuation of Hope" in bold letters, it looks like some sort of exhibition.`
+			choice
+				`	(Check it out.)`
+				`	(Not right now.)`
+					defer
+
+			`	You approach some of the people at the end of the line and ask what's going on. "It's Adelita's exhibition on the evacuation of Hope," says an excited young woman. "It was their first work that really got them famous. The last week of their exhibitions are always free, so this is a great chance to see it if you haven't already."`
+			choice
+				`	"Sure, why not."`
+				`	"Maybe some other time."`
+					defer
+
+			`	"You're going to love it," she says. "No one knows who they really are, but Adelita's work is what inspired me to study journalism. Their mixture of mediums, their eye for story telling... I just love it."`
+			`	You reach the entrance to the exhibition soon enough. Inside is a large room filled with photos, videos, interviews, and other mixed media. They focus on three groups: the refugees of Hope, the Navy that saved them, and the Paradise Worlds that protested the evacuation. At one end of the hangar, there is an installation with nearly a hundred original protest signs from rallies against the Dirt Belt and the Republic Navy; some of them quite crude and tasteless. It's a strong rebuke to see those displayed here on a Paradise Planet.`
```

Comment:
It's an irrational protest. Nevertheless, it is explained in the FW campaign:
> "Here's the best part," says Eyes. "The Navy operates with a certain amount of autonomy. Turns out when they found out about the disaster, they mobilized without even waiting to clear it with Parliament. Which was technically within their mandate, this being a natural disaster. But the Navy is funded by taxes, and the Paradise Planets pay ten times as much in taxes as the Dirt Belt."

> "The Paradise Planets viewed the Navy as their own private security force," says Katya, "so they got really mad when half the Navy suddenly vanished to assist some dirty refugees. Because those admirals gave the order to assist us, millions of lives were saved. And then Parliament fired them."